test/tizen_capi/unittest_tizen_capi.cpp

```diff
@@ -87,6 +89,37 @@ TEST(nntrainer_capi_nnmodel, compile_05_p) {
   EXPECT_EQ(status, ML_ERROR_NONE);
 }
 
+/**
+ * @brief Neural Network Model Compile Test
+ */
+TEST(nntrainer_capi_nnmodel, compile_02_n) {
+  ml_nnmodel_h handle = NULL;
+  int status = ML_ERROR_NONE;
+  status = ml_nnmodel_construct(&handle);
+  EXPECT_EQ(status, ML_ERROR_NONE);
+  status = ml_nnmodel_compile(handle);
+  EXPECT_EQ(status, ML_ERROR_INVALID_PARAMETER);
+  status = ml_nnmodel_destruct(handle);
```

Comment:
Q. Is this okay not to `handle = NULL` inside `ml_nnmodel_destruct` ? If not, Function signature of `ml_nnmodel_destruct` should be changed.

Reply:
It is in ml_nnmodel_destruct.

Reply:
I meant `EXPECT_TRUE(handle == NULL);` would fail after `ml_model_destruct(HANDLE)`. 
I guess it is not a big deal though :)